content/300-guides/200-deployment/201-serverless/500-deploy-to-netlify.mdx

```diff
@@ -128,14 +67,14 @@ netlify deploy
 
 You can now test the deployed application.
 
-## Summary
+## Connection pooling
 
-You have successfully configured continuous deployments and integration with Netlify for an application that uses Prisma.
+When you use a Function-as-a-Service provider, like Netlify, it is beneficial to pool database connections for performance reasons. This is because every function invocation may result in a new connection to your database which can quickly run out of open connections.
 
-Every time you push changes to the `main` branch of the repository, Netlify starts a new production build and deployment. If you push changes to another branch, Netlify starts preview builds and deployments.
+You can use [Accelerate](/data-platform/accelerate) or the [Data Proxy](/data-platform/classic-projects/data-proxy) for connection pooling, to reduce your Prisma Client bundle size, and to avoid cold starts.
```

Comment:
I would recommend omitting Data Proxy in favor of Accelerate.
```suggestion
You can use [Accelerate](/data-platform/accelerate) for connection pooling, to reduce your Prisma Client bundle size, and to avoid cold starts.
```